### PR TITLE
Add troubleshooting for ASIO4ALL

### LINKED ISSUE #732: Routing Jack on Windows Guide Fix for ASIO4ALL
### Discussed in https://github.com/jamulussoftware/jamulus/discussions/2554

<div type='discussions-op-text'>

<sup>Originally posted by **chairbender** March 27, 2022</sup>
EDIT: Sigh, nevermind...it looks like I am still having issues again even after doing this. I swear it fixed my problems on one day, but I guess it isn't consistent...

Hi, I found this page (How to play audio from your PC to Jamulus (using JACK)) very useful for trying to set things up on Windows 10 using ASIO4ALL: https://jamulus.io/kb/2022/01/26/Routing-JACK-Windows.html

However, I ran into a frustrating issue where, when I try to press the  "start" button in QjackCtl with ASIO4ALL as the interface, it (usually) fails to startup and shows an error. After much trial and error, I discovered what fixes it is to go into advanced and change the "Start Delay" to 10 seconds. The default of 2 seems to be insufficient time. Perhaps it is because ASIO4ALL takes longer to start up, as it needs to start up when QjackCtl starts as opposed to other ASIO devices which are constantly running?

Anyway, to save others the trouble, if possible, I would suggest updating the guide I linked to update this section like so:

> Ensure the Sample Rate is set to 48000, the driver is portaudio, Frames/Period is set to something low like 64 or 128. **Go to the "Advanced" tab and ensure "Start Delay" is set to 10 seconds**. You can now close this window with “OK”. Next, click on the Start button in Qjackctl and open Jamulus and go into settings. There select “JackRouter” as device:
</div>

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**
Adds a short section related to ASIO4ALL. 

**Context: Fixes an issue? Related issues**
Fixes: https://github.com/jamulussoftware/jamuluswebsite/issues/732

**Status of this Pull Request**

Ready to be merged

**What is missing until this pull request can be merged?**

KB entry --> no review needed
**Does this need translation?**

NO

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

**Comment by ann0see** [comment]
@chairbender This PR will hopefully be ok to note your error. 

Maintainers: Please merge this PR after approval.

**Review by pljones — APPROVED** [review]

**Pull request merged**
